yaksh/models.py

```diff
@@ -1200,6 +1201,14 @@ def set_grade(self):
             grade = grading_system.get_grade(self.percentage)
             self.grade = grade
             self.save()
+    
+    def get_certificateStatus(self):
+        return self.certificate
+
+    def set_certificateStatus(self):
+        self.certificate = True
```

Comment:
As you are setting a Boolean value to the certificate field, You should use `models.BooleanField`.